lib/validators/helpers.js

```diff
@@ -30,6 +221,92 @@ export function getPreviewText(text, index, length) {
   return previewText;
 }
 
+/**
+ * detrmine whether a conditional-required field should be required
+ * note: exported for testing
+ * @param  {object} field         { type, name, path, schema, value }
+ * @param  {object} componentData
+ * @return {boolean}
+ */
+export function shouldBeRequired(field, componentData) {
```

Comment:
I am not sure what is meant by "conditional-required field should be required". Could you describe a use case?

Reply:
whether that field should be required, based on the config passed in https://claycms.gitbooks.io/kiln/editing-components.html#standard-input-arguments